src/views/Profile/AdminTable/AdminDatePicker.tsx

```diff
@@ -54,39 +139,49 @@ export const AdminDatePicker: React.FC<IMaterialsDate> = ({
           value={[]}
           disableUnderline
           displayEmpty
-          renderValue={() => (
-            <div>
-              {t(langTokens.admin.from)} {parsDate(start)}{' '}
-              {t(langTokens.admin.to)} {parsDate(end)}
-            </div>
-          )}
+          renderValue={() =>
+            !datePickerState.firstSelectedDay.value &&
+            !datePickerState.secondSelectedDay.value ? (
+              <DateRangeOutlinedIcon />
+            ) : (
+              <div>
+                {t(langTokens.admin.from)} {parsDate(start)}{' '}
```

Comment:
I strongly suggest to code this as a single translation like

```
З {{parsedStartDate}} по {{parsedEndDate}}
```

Something like https://github.com/ita-social-projects/dokazovi-fe/blob/develop/src/locales/uk/parts/admin.ts#L79

You see, in different languages structure may be different, e.g. "from" and "to" might be combined into single word between dates or maybe grammar rules requires us to add text _after_ dates, not between. Having this coded in single solid message template gives much more flexibility with no disadvantages.